tests/foreman/cli/test_ansible.py

```diff
@@ -796,6 +798,7 @@ def aap_client(self):
         client.logout()
 
     @pytest.mark.parametrize('auth_type', ['admin', 'non-admin'])
+    @pytest.mark.parametrize('aap_version', ['2.3', '2.5'])
```

Comment:
We are not testing 2.4?

Reply:
We've planned for 2.4 deployment, soon will start working on it, so for now we have 2.3 only